src/ray/raylet/worker_pool.cc

```diff
@@ -596,28 +579,35 @@ void WorkerPool::MarkPortAsFree(int port) {
   }
 }
 
+static bool RuntimeEnvNotEmpty(const std::string &serialized_runtime_env) {
```

Comment:
We currently have `IsRuntimeEnvEmpty()` in `runtime_env_common.h`, does it make sense to use that one?

Reply:
Good! I will reuse yours.